README-CONAN.md

```diff
@@ -140,20 +148,19 @@ $ cmake --build . --config Release
 
 #### Default Profile
 
-When you run conan install for the first time, it will detect and write the default profile ~/.conan/profile/default.  On my Ubuntu system with GCC 4.9, this is:
+When you run conan install for the first time, it will detect and write the default profile `~/.conan/profile/default`.  On a Ubuntu 18.04 system with GCC 7.3, this is:
 
 ```ini
+[build_requires]
 [settings]
 os=Linux
-os_build=Linux
 arch=x86_64
-arch_build=x86_64
 compiler=gcc
-compiler.version=4.9
-compiler.libcxx=libstdc++
+compiler.version=7
```

Comment:
Updated values to a more recent version of Ubuntu and using the needed value of **libcxx**.